### DISCUSSION TITLE
"For detailed information… one would need to refer to the full text of the study."

### DISCUSSION THREAD

**Discussion by ekcomputer** [lifan0127/ai-research-assistant#71] [Q&A] [answered]
Despite my prompt saying "please refer to the full text of the PDF paper for your response," I often get responses that do not answer even my easiest questions, and instead say "For detailed information… one would need to refer to the full text of the study." Does anyone have ideas? What is causing this issue? Do I need to update my prompting? Is there some setting in the software that only allows a ARIA to query the abstract?

**Comment by lifan0127** [comment, marked as answer]
Thanks for raising this issue. Q&A based on PDF full text is currently disabled due to performance and API cost issues. It will be enabled again once the issues are addressed in the near future.

**Reply by ekcomputer** [reply]
Looking forward to it!

**Reply by DanielErben** [reply]
Hello, @lifan0127 - I am curious - any update on this?

**Reply by lifan0127** [reply]
Hi @DanielErben , yes I am still working on it!
